src/components/DeckBuilderDryRunView/index.js

```diff
@@ -207,9 +207,33 @@ class DeckBuilderDryRunView extends React.Component {
           ? 'unit '
           : 'units '}
         on the board.
+        <br />
       </>
     )
   }
+
+  getFrozenEnemiesText = () => {
+    const { frozenEnemiesLevel } = this.props.specifics
+    const frozenStateDescriptionCount = {
+      0: 'no',
+      2: 'a few',
+      3: 'many',
+    }
+    const frozenStateDescription =
+      frozenEnemiesLevel === 4
```

Comment:
Couldn’t `4` happen with a Moment’s Peace?

Reply:
I don't understand what your comment is referring to

Reply:
The Level variable is an indicator of how many frozen enemies Zhevana could destroy. When the RNG mode is UNFRIENDLY for example, all enemy units are placed in different columns/there less enemy units to begin with. When playing Moment's Peace, 4 should not happen since it can freeze at most 3 units in a column.

Reply:
There was still an issue with playing Moment's Peace in FRIENDLY mode - I fixed it with a new commit

Reply:
I see, that makes sense. I thought it was the number of potentially frozen units on the board. Maybe we should rename this variable then. How about `frozenInColumn`?

Reply:
The issue is that counting potential enemy units there are in a column will never work:
+ Once Zhevana destroys a column, we don't know how many frozen enemies there are in the other columns
+ Playing Frosthexers twice in a single turn would allow to freeze a whole column (+2 +2), which is impossible in the game since those 2 can only be frozen when Frosthexers are played between them

I tried implementing something like this yesterday but ran into those issues. That's why this variable represents more of an impression of how many frozen units there are on the board - Hence the text displaying "a few"/"many"/"The whole board"... instead of the precise count as originally intended

Reply:
That's why I changed the name to something more abstract too